framework/components/ecl-comments/ecl-comments.twig

```diff
@@ -49,17 +49,23 @@
       <p class="ecl-paragraph">{{ body }}</p>
     </div>
     {% if operations is defined %}
-    <div class="ecl-comment__operations">
+    <ul class="ecl-listing ecl-comment__operations">
       {% if operations.reply is defined and operations.reply.visible %}
-        <a class="ecl-link ecl-comment__reply" href="{{ operations.reply.href }}">{{ operations.reply.title }}</a>
+        <li class="ecl-list-item">
```

Comment:
https://ec-europa.github.io/europa-component-library/components/detail/ecl-list-items: "List items are intended to be used only in listings component."

Since we don't use the listing component, we don't need this class. `<li>` is enough